plugins/dominant-color-images/hooks.php

```diff
@@ -142,11 +142,11 @@ function dominant_color_img_tag_add_dominant_color( $filtered_image, $context, $
 		$extra_class  = $image_meta['has_transparency'] ? 'has-transparency' : 'not-transparent';
 	}
 
-	if ( ! empty( $data ) ) {
```

Comment:
PHPStan was complaining here:

```
Variable $data in empty() always exists and is always falsy.
```

For some reason switching to a simple boolean check fixed the issue. In any case, using `empty()` is discouraged per https://github.com/WordPress/performance/pull/1091#discussion_r1543908524